integration/README.md

```diff
@@ -0,0 +1,7 @@
+# OpenLineage Integrations
+|Integration|Release Link|Compatible Version|Latest Release|
+|-|-|-|-|
+|Apache Airflow|https://github.com/apache/airflow/releases|1.10.12+, 2.1+ (experimental)|2.2.5|
+|dagster|https://github.com/dagster-io/dagster/releases|0.13.8+|0.14.7|
+|dbt|https://github.com/dbt-labs/dbt-core/releases|0.20+|1.0.4|
+|spark|https://github.com/apache/spark/tags|3.2.1+|3.2.1|
```

Comment:
Please refer to circle-ci config:
https://github.com/OpenLineage/OpenLineage/blob/main/.circleci/workflows/openlineage-java.yml#L12
for Spark versions tested in CI.